### PR TITLE
Follow-up improvements to @waku/app

### PR DESCRIPTION
**Pull request by weboko**
Follow-up thing for #154:
- [x] double check if `execSync` reliable enough - yes, we can safely use the one from nodejs;
- [x] move examples in the repo to dedicated folder and remove usage of `wakuExamples` from `package.json`;
- [x] check import ES modules - checked, we can move to it but I think it is better to refrain from it since this code used as it is by Node;
- [x] configure ci to support this package; - the other thing to do is allow CI to publish new version of the package
- [x] add support similarly to `hardhat`

NB: since I moved examples to dedicated folder - there is huge diff in the PR

### CODE REVIEW

File: ci/Jenkinsfile [hunk 1]
```diff
@@ -78,6 +79,7 @@ def buildExample(example=STAGE_NAME) {
 
 def copyExample(example=STAGE_NAME) {
   sh "mkdir -p build/docs/${example}"
-  sh "cp ${example}/index.html build/docs/${example}/"
-  sh "[ -f ${example}/style.css ] && cp ${example}/style.css build/docs/${example}/ || exit 0"
+  sh "cp examples/${example}/*.html build/docs/${example}/"
+  sh "cp examples/${example}/*.js build/docs/${example}/"
```
Comment (fryorcraken) [inline ci/Jenkinsfile:83]: This will make the Jenkins job fail if no js file is present. Same for css

File: create-waku-app/createApp.js [hunk 2]
```diff
@@ -20,13 +23,28 @@ const init = (name, description, version, supportedExamples) => {
         })
         .option(
             "-t, --template <path-to-template>", 
-            "specify a template for the created project"
+            "specify a template for the created project or you can skip and select template manually"
```
Comment (fryorcraken) [inline create-waku-app/createApp.js:26]: Change to "Or you can use the wizard" as selecting in the wizard is still "manual"

**Review by fryorcraken — COMMENTED** [review]
Would it be better to keep the examples at the root of js-waku-examples repo and just create a new create-waku-app?

Is there a benefit in having the create app in same repo?

I don't remember, so we package the examples code in the create app package? Or do we download the code from the repo upon invocation?

**Comment by weboko** [comment]
> Would it be better to keep the examples at the root of js-waku-examples repo and just create a new create-waku-app?
>
By moving everything I wanted to separate examples from other code and make it easier to identify where examples are, because when they are at the root we need to maintain a dictionary with example name and it's place in the root. We can keep them as before of course. 

> Is there a benefit in having the create app in same repo?
>
The only benefit is that in order to get them in `create-waku-app` we don't need to clone the repo but just take the examples and copy them into npm package. 

> I don't remember, so we package the examples code in the create app package? Or do we download the code from the repo upon invocation?
>
`create-waku-package` works like this: upon releasing it we clone files of the examples and push them into npm registry. If we decide to keep `create-waku-app` then I will need to write some code to clone examples from the repo. I would like to avoid this path since it increases distance between the `create-waku-app` and examples.

**Comment by fryorcraken** [comment]
I tested locally and we are moving away from `light-chat` being the default template. `eth-pm` is selected by default.

Is it possible to have a menu where nothing is selected by default it can just displays (default: light-chat) so that the user can just hit "enter" and get light-chat

And then if they want they can scroll updown

Or having light-chat selected by default in the list. In this case, I think it's still be good to have a label `default`.

The idea being that when a user tries the first time, they don't have to make a decision.

**Comment by weboko** [comment]
> I tested locally and we are moving away from `light-chat` being the default template. `eth-pm` is selected by default.
> 
> Is it possible to have a menu where nothing is selected by default it can just displays (default: light-chat) so that the user can just hit "enter" and get light-chat
> 
> And then if they want they can scroll updown
> 
> Or having light-chat selected by default in the list. In this case, I think it's still be good to have a label `default`.
> 
> The idea being that when a user tries the first time, they don't have to make a decision.

Good spot. I moved the `light-chat` option to the top.

**Review by danisharora099 — APPROVED** [review]
tested locally; lgtm!

have we considered writing the scripts in TS?

**Comment by weboko** [comment]
> have we considered writing the scripts in TS?

I thought of going with it - but it add unnecessary friction for simple package with two files.

**Comment by danisharora099** [comment]
> > have we considered writing the scripts in TS?
> 
> I thought of going with it - but it add unnecessary friction for simple package with two files.

I'd be in the favour of temporary friction writing code that is easy to write more code on top of in the future
cc @weboko

**Pull request merged**